### ISSUE TITLE
Enhancement: Add `keys()` method to Parameter class

### ISSUE THREAD

**Issue by ntjess** [pyqtgraph/pyqtgraph#1526]
According to Python specifications, objects can be `**unpacked` if they possess a `__getitem__` method and a `keys` method. Since `Parameter`s already have a `__getitem__`, the `keys` method will allow such unpacking. I find this quite useful when a single parameter holds all the arguments for a function call:

```python
def myFunc(a=5, b=10, c=15):
  print('called')

params = Parameter(<has children a, b, c>)
myFunc(**params)
```

The current workaround is a somewhat ugly

```python
inputs = {name: param[name] for name in param.names}
```

Simple `keys` implementation:

```python
def keys(self):
  return self.names
```

**Comment by j9ac9k** [comment]
I would certainly approve of this change; if someone wants to put a PR and do a test for it, I'll gladly review/merge 👍

**Comment by j9ac9k** [comment]
Fixed by #1613
